oat-sa/tao-test-runner-qti-fe
# [TAO-9473] WTP-1466 - [ACT] [TTS - FE] Create TTS plugin. Bug fix

Related to : https://oat-sa.atlassian.net/browse/TAO-9473
 
- Do not show plugin button after item render if plugin is not configured
- Handle tab navigation only in Start From Here Mode
- Handle `activate` navigation event to toggle playback
  
#### How to test
 
- Prepare an instance of TAO with taoAct extension
- Prepare a test with APIP data
- Prepare two deliveries with disabled and enabled APIP plugin
- Execute delivery with disabled plugin as a test taker and check that APIP plugin button is not available
- Execute the delivery with enabled plugin and check that tab navigation works only in SFH mode and you can toggle playback with `enter` key in SFH mode
